client/src/components/History/Content/ContentItem.vue

```diff
@@ -94,6 +94,11 @@ const jobState = computed(() => {
     return new JobStateSummary(props.item);
 });
 
+const itemIsRunningInteractiveTool = computed(() => {
+    // If our datset id is in the entrypOintStore it's a running it
+    return entryPointStore.entryPointsForHda(props.item.id).length > 0;
+});
+
 const contentId = computed(() => {
     return `dataset-${props.item.id}`;
```

Comment:
Unrelated, but this is also wrong, and could be a source of DOM conflicts. It should also only be `dataset-` is it's not a collection

Reply:
:Scream: